src/libstore/build/local-derivation-goal.cc

```diff
@@ -580,10 +642,11 @@ void LocalDerivationGoal::startBuilder()
 
         printMsg(lvlChatty, format("setting up chroot environment in '%1%'") % chrootRootDir);
 
-        if (mkdir(chrootRootDir.c_str(), 0750) == -1)
+        // FIXME: make this 0700
```

Comment:
This FIXME is fairly short so I doubt I would know how to interpret it a long time from know. Should we file an issue and link it here?